src/cls/ZPM/Package.cls

```diff
@@ -286,16 +286,30 @@ ClassMethod GetModuleMeta(url As %String) As %DynamicObject
     Set ht.Port = port
   }
   Set ht.Location = tComponents("path")_"master/module.xml"
-    
   $$$ThrowOnError(ht.Get())
   If (ht.HttpResponse.StatusCode'=200) {
     $$$ThrowStatus($$$ERROR(5001,"Unable to download XML "_ht.Location))
   }
-  $$$ThrowOnError(##class(%XML.XPATH.Document).CreateFromStream(ht.HttpResponse.Data, .xpathdoc))
+  Set tmpFolder = $$$FileTempDir
+  Do ##class(%File).CreateDirectory(tmpFolder)
+
+  Set stream=##class(%Stream.FileCharacter).%New()
+  Do stream.LinkToFile(tmpFolder_"/module.xml")
+  Set stream.TranslateTable = "UTF8"
+  Do stream.CopyFrom(ht.HttpResponse.Data)
+  Do stream.%Save()
+
+  set binaryStream=##class(%Stream.FileBinary).%New()
+  do binaryStream.LinkToFile(tmpFolder_"/module.xml")
```

Comment:
binaryStream or stream?

Reply:
binaryStream 
CreateFromStream method requires it. "Use this method to create an instance of an %XML.XPATH.Document from a binary stream. It's important to use a binary stream ..." https://docs.intersystems.com/irislatest/csp/documatic/%25CSP.Documatic.cls?PAGE=CLASS&LIBRARY=%25SYS&CLASSNAME=%25XML.XPATH.Document